site/components/src/main/java/uk/nhs/digital/website/security/stub/HippoUserDetailsServiceStub.java

```diff
@@ -0,0 +1,25 @@
+package uk.nhs.digital.website.security.stub;
+
+import org.onehippo.forge.security.support.springsecurity.authentication.HippoUserDetailsService;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import uk.nhs.digital.intranet.model.IntranetUser;
+
+import java.util.Collections;
+
+public class HippoUserDetailsServiceStub implements HippoUserDetailsService {
```

Comment:
Should this be in the test folder?

Reply:
No. This is used in the local development environment, not in unit tests. It's an actual stub.